addons/mail/static/src/core/common/sound_effects_service.js

```diff
@@ -20,7 +20,7 @@ export class SoundEffects {
                 defaultVolume: 0.15,
                 path: "/mail/static/src/audio/call-invitation",
             },
-            "new-message": { path: "/mail/static/src/audio/new-message", defaultVolume: 0.5 },
+            "new-message": { defaultVolume: 1, path: "/mail/static/src/audio/new-message" },
```

Comment:
This doubles the volume, no other sound has this volume.
Maybe 0.75 instead?

Reply:
No no, it isn't doubled, I re-sampled it but it was twice as low, so I adjusted it to be almost the same.

Reply:
If you wonder why this is halved: I worked with 96khz but I had issues with crackling noise so I now work with 48khz. The resulting sound-sample is 44.1kHz anyway.

Reply:
Hm, actually it's is lower in volume too, but I feel it was too loud. I put some preview in PR message, if you want to compare before / after.